libsgxstep/enclave.c

```diff
@@ -195,6 +195,13 @@ void* get_enclave_ssa_gprsgx_adrs(void)
     return get_enclave_base() + ossa + (cssa * SGX_SSAFRAMESIZE) - SGX_GPRSGX_SIZE;
 }
 
+void set_debug_optin(void) 
+{
+    void *tcs_addr = sgx_get_tcs();
+    uint64_t flags = 0x1;
```

Comment:
not sure what's exactly 0x1 here -- better would be to define some constants in a header file and use these to avoid magic numbers?